### DISCUSSION TITLE
Consulta error al mergear

### DISCUSSION THREAD

**Discussion by MauroBaroni** [frlp-utn-ingsoft/catedra#86] [Q&A]
mientras estaba haciendo mi parte de la actividad con un compañero, el hizo el merge y subio los cambios al main.Al momento de mergear me tiro conflictos, los resolvi desde la rama ,pero me quedo comentado en algunas lineas donde solucione los conflictos msj como <<<HEAD, no me aparecieron hasta que pushie los cambios en main. Fui a mi rama los elimine y subi los commit y los pushie pero en mi rama. Que debería hacer? volver a mergear con main para corregir esos comentarios o tendria que volver atras cambios?

repositorio: https://github.com/MatyRusso22/vetsoft

**Comment by RodrigoJacznik** [comment]
Si, no mergeaste bien la primera vez. 
Volve a mergear con main.
